tests/persistSync.test.tsx

```diff
@@ -0,0 +1,173 @@
+import create from '../src/index'
+import { persist } from '../src/middleware'
+
+const consoleError = console.error
+afterEach(() => {
+  console.error = consoleError
```

Comment:
What is this for? I'm not sure to understand since we're not resigning console.error, and you're neither mocking it.

Reply:
I did copy that from `persistent.test.tsx` and not verify that it's needed, good catch. I can be removed, right? Might make sense to remove it from `persistent.test.tsx` as well, I guess.

Reply:
Actually this concerns all test files. It's not apparent to me why `console.error` gets mocked. Does zustand write to `console.error`, and the mocks where once verifying that? Address in a separate PR?

Reply:
cc @dai-shi

Reply:
@barbogast

> Actually this concerns all test files.

Totally. They are not well maintained, I'd say... 😅  Happy to take PRs for improvements. Please address in a separate PR.